### PR TITLE
Move sweeping parameters to a file

### PR DESCRIPTION
**Pull request by mamikonyana**
@deepakn94

### CODE REVIEW

File: bench/execute_workflows.py [hunk 1]
```diff
@@ -193,12 +192,17 @@ def parse_args():
   parser = argparse.ArgumentParser()
   parser.add_argument('--configurations-file',
                       type=argparse.FileType('r'),
-                      default='all_configuration_parameters.json',
+                      default='conf/all_configuration_parameters.json',
+                      help='File with a list of configuration parameters')
+  parser.add_argument('--sweeping-parameters-file',
+                      type=argparse.FileType('r'),
+                      default='conf/sweeping_parameters.json',
                       help='File with a list of configuration parameters')
```
Comment (deepakn94) [inline bench/execute_workflows.py]: Nitpick: Might want to specify the type of parameters in the help string.

**Comment by deepakn94** [comment]
Looks good to me.

**Comment by mamikonyana** [comment]
modified the help string to be more clear.

**Comment by deepakn94** [comment]
Did you push?

**Comment by mamikonyana** [comment]
I did now

**Pull request merged**